phpseclib/File/ASN1.php

```diff
@@ -23,8 +23,9 @@
 
 namespace phpseclib3\File;
 
-use DateTime;
```

Comment:
I opened up https://github.com/phpseclib/phpseclib/blob/master/phpseclib/File/ASN1.php and did Ctrl + F for Base64 and it doesn't look like it's actually used in that file?